### DISCUSSION TITLE
[운영체제 4일차] CPU 스케쥴링

### DISCUSSION THREAD

**Discussion by seongeun42** [Shortudy-10th/tech-for-developer#87] [오늘의 주제]
스케쥴링 알고리즘 제외한 CPU 스케쥴링 질문 2인조 1개, 3인조 2개

**Comment by watty-exclamationMark** [comment]
CPU 스케쥴러는 무엇인가요?

**Comment by kjc6735** [comment]
CPU 스케줄링의 성능 척도에는 어떤 것들이 있나요?

**Comment by Jihn0118** [comment]
운영체제가 스케줄링 결정을 내리는 시점과 방식에 대해 설명해주세요.

**Comment by seongeun42** [comment]
스케줄링의 목적과 저, 중, 고수준 단계에 대해 말해주세요.

**Comment by seongeun42** [comment]
스케줄러가 어떤 프로세스에 우선적으로 CPU를 할당할지 스케줄링 시 고려하는 사항에 대해 말해주세요.
